packages/mobile/src/screens/chats-screen/ChatsScreen.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from 'react'
+
+import { chatActions, chatSelectors } from '@audius/common'
+import { View } from 'react-native'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { Screen, Text } from 'app/components/core'
+
+export const ChatsScreen = () => {
+  const dispatch = useDispatch()
+  const chats = useSelector(chatSelectors.getChats)
+  const id = 'clbyghuy800003bat0th2ivy5'
```

Comment:
this cool just merging? like its not an important key or something?

Reply:
nah just a hardcoded chat id for testing